juntagrico/views_subscription.py

```diff
@@ -446,42 +440,38 @@ def change_nickname(request, subscription_id):
 
 
 @login_required
-def order_shares(request):
+def manage_shares(request):
     if request.method == 'POST':
-        referer = request.POST.get('referer')
         try:
             shares = int(request.POST.get('shares'))
             shareerror = shares < 1
         except ValueError:
             shareerror = True
         if not shareerror:
             create_share(request.user.member, shares)
-            return redirect('{}?referer={}'.format(reverse('share-order-success'), referer))
+            return redirect(reverse('manage-shares'))
     else:
         shareerror = False
-        if request.META.get('HTTP_REFERER')is not None:
-            referer = request.META.get('HTTP_REFERER')
-        else:
-            referer = 'http://' + Config.adminportal_server_url()
+    member = request.user.member
+    shares = member.share_set
     renderdict = get_menu_dict(request)
+    overflow_list = [shares.count()]
+    overflow_list.append(member.subscription_future.share_overflow) if member.subscription_future is not None else 0
```

Comment:
why not make a normal if statement a line above?

Reply:
i like one line ifs but you are right here it takes the concept to far